.evergreen/install-dependencies.sh

```diff
@@ -4,6 +4,10 @@ set -o errexit  # Exit the script with error if any of the commands fail
 
 install_extension ()
 {
+   # Workaround to get PECL running on PHP 7.0
```

Comment:
Would it make sense to guard this in a conditional for `PHP_VERSION` starting with "7.0."?

https://stackoverflow.com/a/2172367/162228 suggests you could use `if [[ $PHP_VERSION == 7.0.* ]]`

Reply:
Despite me trying to use `bash` as a shell, I couldn't get evergreen to play nice:
```
[2020/09/22 08:21:34.150] + [[ 7.0.32 == 7.0.* ]]
 [2020/09/22 08:21:34.150] /data/mci/b4ada7bfe7e6167ba91c9754dd42a065/src/.evergreen/install-dependencies.sh: 8: /data/mci/b4ada7bfe7e6167ba91c9754dd42a065/src/.evergreen/install-dependencies.sh: [[: not found
```

Since this has no negative impact on other PHP versions, I'd use this unconditionally, especially since 7.0 is the next PHP version we'll drop support for (although no timeline for this exists).